crates/op-hardforks/src/constants.rs

```diff
@@ -0,0 +1,53 @@
+//op mainnet
+/// Bedrock hardfork activation block
+pub const OP_BEDROCK_MAINNET_BLOCK: u64 = 105235063;
+/// Canyon hardfork activation timestamp
+pub const OP_CANYON_MAINNET_TIMESTAMP: u64 = 1704992401;
```

Comment:
```suggestion
pub const OP_CANYON_MAINNET_TIMESTAMP: u64 = 1_704_992_401;
```

let's make these numbers easier to read like this